ngraph/core/src/validation_util.cpp

```diff
@@ -224,8 +224,45 @@ PartialShape ngraph::infer_windowed_reduction_output_shape(const Node* node,
     return output_shape;
 }
 
+void ngraph::validate_conv_params_spatial_dimensions(const Node* node,
+                                                     const size_t num_spatial_dims,
+                                                     const op::PadType auto_pad,
+                                                     Strides& strides,
+                                                     Strides& dilations,
+                                                     CoordinateDiff& pads_begin,
+                                                     CoordinateDiff& pads_end)
+{
+    if (strides.size() == 0)
+    {
+        strides = Strides(num_spatial_dims, 1);
```

Comment:
I really don't like that this function does 2 things: validates and provides default value. Name suggests it should only validate, so I would expect that if values are 0 then it will throw. Please rethink if it's a good design, if you don't see a better alternative then ok, let's leave it as is.